specify/specify7
# Improve backend query export names

> Yes, we should improve the back-end file name. But, Caroline created a separate issue for that, so it doesn't necessarily have to be fixed as part of this PR - https://github.com/specify/specify7/issues/3525

Addresses https://github.com/specify/specify7/pull/3476#issuecomment-1560024885 and part of #3525 

Follows the more user-friendly Query Export file name introduced in #3476. 
All query export files should now be in the following format: `QueryName - Weekday (abbreviated) Month (abbreviated) Day Year`
Example: `New Query - Wed Jun 07 2023`

To Test:
Without selecting any rows:
- Export a Query to csv
  - Ensure the file name has the correct format (which should be similar to the file name when selected rows are exported)
- Export a Query to kml 
  - Ensure the file name has the correct format
  - The `Export to KML` button is only visible when the Query contains Coordinate data (like Longitude1 and Latitude1)
  - No testing needs to be done regarding exporting selected rows to kml: this is not currently supported (#3589)
